r/R/dplyr-datetime-helpers.R

```diff
@@ -166,6 +166,18 @@ build_formats <- function(orders) {
   orders <- gsub("[^A-Za-z]", "", orders)
   orders <- gsub("Y", "y", orders)
 
+  invalid_orders <- nchar(gsub("[ymdqIHMS]", "", orders)) > 0
```

Comment:
You might need to make this more permissive if you want to include the additional formats.

Reply:
I've made this into a different check. Basically the format needs to be longer than 6 chars to be considered valid (e.g.: "ymd" -> "%Y%m%d" is 6 chars long and is probably the shortest format we have).

Reply:
I've made this into a different check. Basically the format needs to be longer than 6 chars to be considered valid (e.g.: "ymd" -> "%Y%m%d" is 6 chars long and is probably the shortest format we have).